pkg/catalog/catalog_test.go

```diff
@@ -83,4 +82,19 @@ var _ = Describe("Catalog", func() {
 			})
 		})
 	})
+
+	Describe("GetGreaterThan", func() {
+		It("lists all available versions which are greater than the current version in descending order", func() {
+
+			profiles := []profilesv1.ProfileDescription{{Name: "foo", Version: "v0.1.0", Description: "install foo"}, {Name: "foo", Version: "v0.2.0"}, {Name: "foo"}, {Name: "foo", Version: "v0.3.0"}, {Name: "foo"}}
+			c.Update(catName, profiles...)
+
+			Expect(c.GetGreaterThan(catName, "foo", "v0.1.0")).To(Equal(
+				[]profilesv1.ProfileDescription{
+					{Name: "foo", Version: "v0.3.0", CatalogSource: catName},
+					{Name: "foo", Version: "v0.2.0", CatalogSource: catName},
+				},
+			))
+		})
+	})
```

Comment:
I feel burned by my failed attempt at this in the past. shall we expand the test to have:
a version "older" than `v0.1.0` (and see that not returned) and a version newer than `v0.1.0` belongs to a different profile? Just to cover all the edge cases. This way I can't break it in the future :smile:

Reply:
Hahahaha, yeah sure. :)